paramiko/auth_handler.py

```diff
@@ -297,7 +300,6 @@ def _parse_service_accept(self, m):
                     maj_status = m.get_int()
                     min_status = m.get_int()
                     err_msg = m.get_string()
-                    lang_tag = m.get_string()  # we don't care!
```

Comment:
I guess this can be dropped because an exception is immediately raised?

(It's worth pointing out the non-whitespace changes for a more careful review.)